docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+# NB: You must have manually installed ArkCase inside a Docker
```

Comment:
would you put a Dockerfile into this git repository, or would it be stored somewhere else?  How would somebody go about manually installing ArkCase inside a Docker container?

Reply:
As I documented in Confluence, I can't write a Dockerfile. The reason is that Docker just allows basic operations while building the image, and especially doesn't run any daemons. But the Ansible scripts expect systemd to be running, along with a number of services.

Reply:
OK... how would somebody build the docker image manually?

Reply:
There are step-by-step instructions there: https://arkcase.atlassian.net/wiki/spaces/DO/pages/6422529/Notes+about+Dockerizing+ArkCase

Reply:
@fabricetriboix if you add this link into the docker-compose.yml then we can merge this one.